src/components/layer-panel/layer-panel.vue

```diff
@@ -29,23 +30,19 @@ function onDisplayCatalog() {
 </script>
 
 <template>
-  <div data-cy="layerPanel" class="flex flex-col h-full pt-1.5">
+  <base-panel-layout
```

Comment:
Could it make sense to call this `side-panel-layout`?

Reply:
or side-panel-base